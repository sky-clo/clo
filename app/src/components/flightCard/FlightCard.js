import React from "react";
import { Link } from "react-router-dom";

import styles from "./FlightCard.module.scss";

export default function FlightCard(props) {
  return (
    <Link to={props.href} className={styles.flightCard}>
      <p className={styles.title}>{props.title}</p>
      <p className={styles.price}>{props.price}</p>
    </Link>
  );
}
