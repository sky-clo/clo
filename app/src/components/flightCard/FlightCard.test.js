import React from "react";
import { shallow } from "enzyme";

import FlightCard from "./FlightCard";

describe("FlightCard", () => {
  it("should match snapshot", () => {
    const render = shallow(
      <FlightCard
        title="London Stansted to Split"
        time="6h 9m"
        price="Â£41"
        href="/location"
      />
    );
    expect(render).toMatchSnapshot();
  });
});
