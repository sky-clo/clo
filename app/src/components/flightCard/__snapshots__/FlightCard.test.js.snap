// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlightCard should match snapshot 1`] = `
<a
  className="flightCard"
  href="/location"
>
  <p
    className="title"
  >
    London Stansted to Split
  </p>
  <p
    className="time"
  >
    6h 9m
  </p>
  <p
    className="price"
  >
    Â£41
  </p>
</a>
`;
